main.py

```diff
@@ -103,15 +104,78 @@ def retrieve(namespace, box_id):
 
     @staticmethod
     @rest('v1/<namespace>/<box_id>', methods=['DELETE'])
-    def delete(namespace, box_id):
+    def rest_delete(namespace, box_id):
         """Delete a box from a namespace."""
         backend = FileSystem()
         result = backend.delete(namespace, box_id)
         if result:
-            return jsonify({"response": "Box deletedd"}), 202
+            return jsonify({"response": "Box deleted"}), 202
 
         return jsonify({"response": "Unable to complete request"}), 500
 
+    @staticmethod
+    @listen_to('kytos.storehouse.create')
+    def event_create(event):
+        """Create a box in a namespace based on an event."""
+        box = Box(event.content['data'], event.content['namespace'])
+        backend = FileSystem()
+        backend.create(box)
+
+        try:
+            event.content['callback'](box, error=False)
```

Comment:
You dont know here if error is false, right ? 
120-133 should be on a try...except...